warc_gpt/commands/ingest.py

```diff
@@ -40,6 +42,10 @@ def ingest() -> None:
     total_records = 0
     total_embeddings = 0
 
+    encoding_timings = []
+    multiplier = 1.1
```

Comment:
Very minor thing: this variable could use a more descriptive name given its scope